Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+alloy = "0.5.4"
```

Comment:
Why is alloy a dependency?

Reply:
mistake, wanted to use it for something
never needed it